docs/contributors/git-workflow.md

```diff
@@ -1,13 +1,13 @@
 # Git Workflow
 
 A good workflow for new contributors to follow is listed below:
-- Fork Gutenberg repository
-- Clone forked repository
-- Create a new branch
-- Make code changes
-- Commit code changes within the newly created branch
-- Push branch to forked repository
-- Submit Pull Request to Gutenberg repository
+- Fork the Gutenberg repository.
+- Clone the forked repository.
+- Create a new branch.
+- Make code changes.
+- Commit the code changes within the newly created branch.
+- Push the branch to the forked repository.
+- Submit a Pull Request to the Gutenberg repository.
 
```

Comment:
Can we here add a Quick Guide approach? Using the above list and adding in the terminal commands used,